__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const app = require("../app.js");
+const request = require("supertest");
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed.js");
+const db = require("../db/connection.js");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/topics", () => {
+  describe("GET", () => {
+    it("responds with an array of objects that have slug and description properties", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body: {topics} }) => {
+          expect(Array.isArray(topics)).toBe(true);
```

Comment:
If the array is empty, the .forEach will be skipped! That means you _could_ have a false positive. How could you prove that it is not empty beforehand?